src/utils/validator.js

```diff
@@ -0,0 +1,36 @@
+const validator = require('validator')
+
+module.exports.signup = (email, password, phone, first_name, last_name, address) => {
+    const errors = {};
+    if (phone === '') {
```

Comment:
Why don't you also check if phone number is a valid phone number (with length and if it's actually numbers)?

Reply:
This would be nice in the case of a Nigerian phone number that would mean the number of digits should be 11. this can be easily be done with regex values

Reply:
I meant making sure they don't send back alphanumeric chars or too short or long numbers. From my experience phone numbers are usually not longer than 20 digits and not shorter than 3 digits.

Reply:
Fixed